preprod.healdata.org/metadata/aggregate_config.json

```diff
@@ -442,14 +442,19 @@
 			"filters": {
 				"datasets": [
 					"laws-regulating-administration-of-naloxone",
-					"drug-induced-homicide",
```

Comment:
Why delete?

Reply:
@xritter1 Two things: (1) forgot to add new studies to datasets list. (2) Then, I deleted the "unused" dataset endpoints from list. That is, only the laws-regulating-administration-of-nalaxone are currently used (see the one per item in config). The new dataset endpoints in list were used to grab the metadata associated with the files (ie other per item records). --- so, I thought it would be easier to follow what is being used currently by syncing the datasets list with the per item list. Does that make sense?

Reply:
Sounds good, thanks! Let me know when this PR is ready for review.

Reply:
@xritter1  great, its ready now! (btw -- everything else remains the same -- just the dataset list is what i changed)

Reply:
Yes I can see that from the latest commit. thanks!